mscore/editstyle.cpp

```diff
@@ -38,12 +38,11 @@ namespace Ms {
 //---------------------------------------------------------
 
 EditStyle::EditStyle(Score* s, QWidget* parent)
-   : QDialog(parent)
+   : cs(s), QDialog(parent)
```

Comment:
This does cause a new compiler warning in MinGW:

In file included from C:\Users\Jojo\Documents\GitHub\MuseScore\mscore\editstyle.cpp:16:0:
C:\Users\Jojo\Documents\GitHub\MuseScore\mscore\editstyle.h: In constructor 'Ms::EditStyle::EditStyle(Ms::Score*, QWidget*)':
C:\Users\Jojo\Documents\GitHub\MuseScore\mscore\editstyle.h:59:14: warning: 'Ms::EditStyle::cs' will be initialized after [-Wreorder]
       Score* cs;
              ^~
C:\Users\Jojo\Documents\GitHub\MuseScore\mscore\editstyle.cpp:41:27: warning:   base 'QDialog' [-Wreorder]
    : cs(s), QDialog(parent)
                           ^
C:\Users\Jojo\Documents\GitHub\MuseScore\mscore\editstyle.cpp:40:1: warning:   when initialized here [-Wreorder]
 EditStyle::EditStyle(Score* s, QWidget* parent)

No idea though how to fix it

Reply:
Just reorder QDialog constructor call and `cs` initialization:
`: QDialog(parent), cs(s)`

These two do not depend on each other so in this case this warning doesn't point to any real issue.

Reply:
Thanks, see #5548